packages/db/postgres/tables.sql

```diff
@@ -25,6 +33,17 @@ CREATE TABLE IF NOT EXISTS public.user
   updated_at      TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS public.user_tag
+(
+  id              BIGSERIAL PRIMARY KEY,
+  user_id         BIGINT                                                        NOT NULL REFERENCES public.user (id),
+  tag             user_tag_type                                                 NOT NULL,
+  -- tag_value is useful for certain tags like STORAGE_LIMIT e.g. tag="STORAGE_LIMIT", tag_value="1TB"
```

Comment:
I am a bit concerned here. Why a text with "1TB" string?

Two things here:
- text for being a generic value is ok, but I would prefer to see a type enum with that type the value is and a text value that you can parse as stated by type
- We stored user used storage as bytes number in the DB. We should follow same pattern here and use bytes number value.

With above, I would expect the admin dashboard to be able to receive and parse tag values such as 1TB, but convert it to bytes instead. Everything would be handled in bytes within API logic instead of conversations for every single validation of used storage

Reply:
Given that `user_tag` is meant to be a very general table, it follows that potential values for tags may range from all sorts of different data types.  `PINNING` doesn't need a value, `STORAGE_LIMIT` uses bytes as you mentioned: but what about future tags?  To make the tag value `bytes` assumes that all future tags will have byte values.  Also thinking about this from an admin perspective, I wonder if a user will want to type `1000000000000` when they really mean `1TB`?

Reply:
@adamalton i see.  yea, i like your proposal and feel it will fit in and be generic enough for all use cases i can think of moving forward.  can you see any downsides to this approach?  i do wonder if being too general here is a limiting factor, but for now it seems good.  not sure how i missed the proposal from @vasco-santos to have another column to denote the type but that makes perfect sense to me.

Reply:
> can you see any downsides to this approach?
@jsdevel I think that this approach gives us the best balance that we're going to get between flexible/generic and not too crazy. The only thing that ran through my mind was that casting values from text to their declared type (e.g. `parseInt(text_value)`) feels like it's veering a little bit towards `eval(untrusted_value_from_db)`! But I think it's safely a good distance away from that yet!

Reply:
Thanks @adamalton to put that code snippet together! That was exactly my thoughts